src/exporter/DaemonMetricCollector.cc

```diff
@@ -147,15 +147,13 @@ void DaemonMetricCollector::dump_asok_metrics() {
           std::string counter_name = perf_group + "_" + counter_name_init;
           promethize(counter_name);
 
-          if (counters_labels.empty()) {
-            auto labels_and_name = get_labels_and_metric_name(daemon_name, counter_name);
-            if (labels_and_name.first.empty()) {
-              dout(1) << "Unable to parse instance_id from daemon_name: " << daemon_name << dendl;
-              continue;
-            }
-            labels = labels_and_name.first;
-            counter_name = labels_and_name.second;
+          auto labels_ = get_extra_labels(daemon_name);
```

Comment:
An automatic variable ending with an underscore is weird -- a trailing underscore designates class members in some style guides:
```suggestion
          auto extra_labels = get_extra_labels(daemon_name);
```